reference/3.0/Microsoft.PowerShell.Security/Set-AuthenticodeSignature.md

```diff
@@ -10,99 +10,109 @@ title:  Set-AuthenticodeSignature
 # Set-AuthenticodeSignature
 
 ## SYNOPSIS
-
-Adds an Authenticode signature to a Windows PowerShell script or other file.
+Adds an Authenticode signature to a PowerShell script or other file.
```

Comment:
I'd provide a link out to the overview for an Authenticode signature: https://docs.microsoft.com/en-us/windows-hardware/drivers/install/authenticode

Reply:
Added